implementations/rust/ockam/ockam_api/src/cli_state/storage/journeys_repository_sql.rs

```diff
@@ -282,14 +296,17 @@ mod test {
         assert_eq!(actual, Some(host_journey2));
 
         Ok(())
+        }).await
     }
 
     /// This test checks that we can store project journeys with a previous / next relationship
     #[tokio::test]
     async fn test_several_project_journeys() -> Result<()> {
-        let repository = create_repository().await?;
+        with_application_dbs(|db| async move {
+            let repository: Arc<dyn JourneysRepository> =
+                Arc::new(JourneysSqlxDatabase::new(db));
 
-        // create and store a the first host journey
+            // create and store a the first host journey
```

Comment:
Apparently cargo fmt gave up )

Reply:
Indeed, I had to do the formatting everywhere `with_dbs/with_application_dbs` was called and I forgot that spot.